### ISSUE TITLE
All WMCore service configuration files should use test AMQ broker for preproduction cluster

### ISSUE THREAD

**Issue by vkuznet** [dmwm/WMCore#11582]
**Impact of the bug**
Correctly use AMQ brokers for CMS data in MONIT

**Describe the bug**
We found that current WMCore configuration files are using production AMQ broker on preproduction cluster. Currently there is one variable
```
AMQ_HOST_PORT = [('cms-mb.cern.ch', 61313)]
```
which is used in WMCore config files both in production and preproduction branches of corresponding gitlab repository. Therefore, when we deploy configuration to corresponding k8s cluster we incorrectly use production broker for preproduction deployment.

**How to reproduce it**
Look-up `config.py` or similar file in preprod branch of gitlab repository.

**Expected behavior**
According to discussion in [RQF2289906](https://cern.service-now.com/service-portal?id=ticket&table=u_request_fulfillment&n=RQF2289906) we should use the following conventions (here is an example of MSPileup service):
```
- for testbed
AMQ_HOST_PORT = [('cms-test-mb.cern.ch', 61313)]
data.topic_amq = "/topic/cms.ms-pileup"
data.producer = "cms-ms-pileup-test"
data.host_port_amq = AMQ_HOST_PORT

- for production
AMQ_HOST_PORT = [('cms-mb.cern.ch', 61313)]
data.topic_amq = "/topic/cms.ms-pileup"
data.producer = "cms-ms-pileup"
data.host_port_amq = AMQ_HOST_PORT
```
In other words:
- we should distinguish host name for production (`cms-mb.cern.ch`) and preproduction (`cms-test-mb.cern.ch`) clusters. The topic may or may not be the same, and we should properly define our producer, i.e. use `-test` suffix for test producers.

**Additional context and error message**
- [RQF2289906](https://cern.service-now.com/service-portal?id=ticket&table=u_request_fulfillment&n=RQF2289906) MONIT ticket for MSPileup
- [gitlab-216](https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/216) ticket for MSPIleup in preprod cluster
- [gitlab-215](https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/215) ticket for MSPIleup in prod cluster

**Comment by vkuznet** [comment]
I provided all required changes in the following PR: https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/217 Once reviewed and merged the action item will be to deploy new secrets in testbed and restart services.

**Comment by amaltaro** [comment]
@vkuznet Valentin, my understanding is that this ticket has been taken care of with your services_config merge request above, and testbed services have also been updated with these changes last Wednesday, May 25th. I guess we can close this ticket, but in case I missed anything, please reopen it. Thank you for these fixes btw!
